core/start-dev-env.sh

```diff
@@ -55,16 +55,16 @@ while true; do
 		# Use docker compose to start the environment but with the modified override file(s)
 		echo -e "\nStarting the ${MODE} environment with the following command:\n"
 
-		echo -e "docker compose --profile php -f docker-compose.yml "${docker_compose_args[@]}" "${extra_compose_args}" up -d "${@:$number_of_dev_envs}"\n"
-		docker compose --profile php -f docker-compose.yml ${docker_compose_args[@]} ${extra_compose_args} up -d "${@:$number_of_dev_envs}"
+		echo -e "docker compose --profile oidc -f docker-compose.yml "${docker_compose_args[@]}" "${extra_compose_args}" up -d "${@:$number_of_dev_envs}"\n"
```

Comment:
@quartje is it possible to do this in order to be able to push from Manage to EB and don't get an exception in Manage?
Or is this the wrong way to achieve this?

Reply:
Manage pushes to both oidc and eb. This would fix that issue. 
In an ideal world, Manage would be configured dynamically (so --profile php would config manage only to push to eb, not to oidc).